src/main/java/frc/robot/RobotContainer.java

```diff
@@ -329,45 +329,4 @@ public Command scoreCoral() {
             elevator.setHeight(ElevatorConstants.STOW_METER).withTimeout(0.5))
         .withName("Score Coral");
   }
-
-  public Command scoreCoralL4() {
-    return Commands.sequence(
-            arm.setAngle(ArmConstants.L4_SCORE_POSITION).withTimeout(1),
-            driveCommand.driveBackward(1).withTimeout(0.2),
-            arm.setAngle(ArmConstants.ARM_STOW_ANGLE).withTimeout(0.5),
```

Comment:
It might just be easier and better practice to put constants for the timeout values instead

just do that for the `prepScoreCoral` and `scoreCoral` commands btw